frontend/src/components/pages/ErrorPage.js

```diff
@@ -3,13 +3,37 @@ import Component from '../../core/Component.js';
 export default class ErrorPage extends Component {
   _title;
   _params;
-  constructor() {
+  constructor(params = null) {
     super(document.querySelector('#app'));
     this._title = 'Error';
+    if (params != null) this._params = params;
+    else {
+      this._params = {
+        code: '404',
+        msg: 'Not found',
+      };
+    }
```

Comment:
```suggestion
  constructor(params = {}) {
    super(document.querySelector('#app'));
    this._title = 'Error';
    this._params = {
      code: '404',
      msg: 'Not found',
      ...params,      
    }
  }
```

이렇게 바꾸는건 어때요? 인자를 선택적으로 넘길 수 있게 됩니다.
이러한 방식으로 `new ErrorPage({msg:'User Not Found'})` 이렇게 생성해도 code 의 default 값이 404가 될 수 있는 선택적 인자 전달이 가능케 합니다.